opengrok-web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -56,11 +56,8 @@
     <filter-mapping>
         <filter-name>ExpiresHalfHourFilter</filter-name>
         <url-pattern>/history/*</url-pattern>
```

Comment:
why not history ?

Reply:
history does not include mast.jsp so this does not affect it anyway

Reply:
Does this mean that history page will not display any messages ?

Reply:
No. It means it will work like now, with sometimes inconsistent view as mentioned in #2405 because of the 30 minutes caching interval. This is also a subject to change if desired, but in different PR.